hazelcast/src/main/java/com/hazelcast/cache/impl/ICacheRecordStore.java

```diff
@@ -505,4 +505,10 @@ CacheRecord merge(CacheEntryView<Data, Data> cacheEntryView, CacheMergePolicy me
      * @return the used {@link CacheRecord} if merge is applied, otherwise {@code null}
      */
     CacheRecord merge(MergingEntryHolder<Data, Data> mergingEntry, SplitBrainMergePolicy mergePolicy);
+
+
+    /**
+     * @return partition id of this store
```

Comment:
Should be `partition ID` (spelling has been unified some time ago).

Reply:
fixed